hydra-node/exe/tx-cost/TxCost.hs

```diff
@@ -82,8 +82,8 @@ computeInitCost = do
 
 computeCommitCost :: IO [(NumUTxO, TxSize, MemUnit, CpuUnit, Lovelace)]
 computeCommitCost = do
-  interesting <- catMaybes <$> mapM compute [1, 2, 3, 5, 10, 50, 100]
-  limit <- maybeToList . getFirst <$> foldMapM (fmap First . compute) [500, 499 .. 101]
+  interesting <- catMaybes <$> mapM compute [1, 2, 3, 5, 10]
+  limit <- maybeToList . getFirst <$> foldMapM (fmap First . compute) [100, 99 .. 11]
```

Comment:
were these values got reduced due to changes in the scripts?

why u say the cost of this is sad now?

Reply:
We were not actually testing anything else than `1`. Sad because the maximum seems to be around `13` ada-only UTxO`, i.e. it's very expensive.